components/theme-provider/radix-themes.provider.tsx

```diff
@@ -4,7 +4,7 @@ import { Theme } from '@radix-ui/themes';
 
 export function RadixThemesProvider({ children }: PropsWithChildren) {
   return (
-    <Theme accentColor="amber" appearance="dark" grayColor="sand">
+    <Theme accentColor="gray" appearance="dark">
```

Comment:
`amber` was a bit too bright... I guess it's safer to go with a monochrome schema then decide on colors.